packages/scaffolding-cli/templates/src/ssr/lib/auth-cookies.ts

```diff
@@ -0,0 +1,46 @@
+import {serialize} from "cookie"
+import Redis from "ioredis"
+import connectRedis from "connect-redis"
+import uid from "uid-safe"
+import session from "express-session"
+
+export const setPassportSessionCookie = (isRedisEnabled, conf) => {
+    // Express session for Auth
+    let RedisStore = connectRedis(session)
+    let redisClient = null
+
+    if (isRedisEnabled) {
+        redisClient = new Redis({
+            port: conf.REDIS_PORT, // Redis port
+            host: conf.REDIS_HOST, // Redis host
+        })
+    }
+
+    const sessionConfig = {
+        store: !isRedisEnabled
+            ? null
+            : new RedisStore({
+                  client: redisClient,
+                  logErrors: error => console.warn("session error: ", error),
+              }),
+        secret: uid.sync(18),
+        cookie: {
+            maxAge: 86400 * 1000, // 24 hours in milliseconds
```

Comment:
Should this be configurable?

Reply:
We can make it configurable. But this is the recommended setup.